packages/amplify-graphql-searchable-transformer/src/graphql-searchable-transformer.ts

```diff
@@ -429,8 +429,9 @@ function getTable(context: TransformerContextProvider, definition: ObjectTypeDef
 
 function getNonKeywordFields(def: ObjectTypeDefinitionNode): Expression[] {
   const nonKeywordTypeSet = new Set(nonKeywordTypes);
+  const datastoreReservedFields = ['_version', '_deleted', '_lastChangedAt'];
```

Comment:
Can we store this in constants scope? So that, this can be reused in other places if required.